### PR TITLE
Expose foundation page

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

File: en-US/foundation/index.lang [hunk 1]
```diff
@@ -0,0 +1,142 @@
+## NOTE: demo page at https://www-dev.allizom.org/foundation
+## URL: https://www-dev.allizom.org/%LOCALE%/foundation
+
+
+;The Mozilla Foundation
+The Mozilla Foundation
+
+
+;Cybersecurity
+Cybersecurity
```
Comment (flodolo) [inline en-US/foundation/index.lang]: I see this in bedrock/foundation/templates/foundation/index.html but it doesn't seem used anywhere?

File: en-US/foundation/index.lang [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+
+;Net Neutrality, Encryption, Surveillance, Copyright Law, Cybersecurity
+Net Neutrality, Encryption, Surveillance, Copyright Law, Cybersecurity
```
Comment (flodolo) [inline en-US/foundation/index.lang]: Same for this one

File: en-US/foundation/index.lang [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+
+;Mozilla Corporation, a wholly owned subsidiary of the Foundation, makes consumer Internet products that advance the same values outlined in the Mozilla Manifesto.
+Mozilla Corporation, a wholly owned subsidiary of the Foundation, makes consumer Internet products that advance the same values outlined in the Mozilla Manifesto.
+
+
+;Firefox, our flagship product, gives increased choice, privacy and security to 100s of millions of people around the world. Product exploration initiatives like our ‘connected devices’ program aim to bring more mainstream products with Mozilla values into the world. Technology initiatives like Rust and Mozilla’s online game centre aim to spread new thinking about open technology across the whole of the Internet industry.
+Firefox, our flagship product, gives increased choice, privacy and security to 100s of millions of people around the world. Product exploration initiatives like our ‘connected devices’ program aim to bring more mainstream products with Mozilla values into the world. Technology initiatives like Rust and Mozilla’s online game centre aim to spread new thinking about open technology across the whole of the Internet industry.
```
Comment (flodolo) [inline en-US/foundation/index.lang]: Peculiar spelling for centre.

File: es-ES/foundation/index.lang [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+The real power behind Mozilla is a global community of 10,000s of volunteers, allies and partners. Members of the Mozilla community participate by doing everything from contributing code to Firefox and <a href="%(url_learning)s">teaching digital literacy</a> to translating our software, organizing advocacy campaigns and writing the world's most referenced <a href="%(url_mdn)s">web developer site</a>.
+
+
+;Get Involved
+Get Involved
+
+
+;Join us and help protect the internet as a global public resource. To get started, <a href="%(url_community)s">join our community</a> or <a href="%(url_donation)s">make a donation</a>.
+Join us and help protect the internet as a global public resource. To get started, <a href="%(url_community)s">join our community</a> or <a href="%(url_donation)s">make a donation</a>.
```
Comment (flodolo) [inline es-ES/foundation/index.lang]: This should probably be Internet, not internet.

File: en-US/foundation/index.lang (hunk 1 above)
Comment (TheoChevalier) [inline en-US/foundation/index.lang]: Correct, strings for the search experiment, let’s remove those two

**Pull request merged**